libs/lexer/src/Lexer.php

```diff
@@ -169,19 +182,21 @@ private function run(ReadableInterface $source, int $offset): \Generator
                 continue;
             }
 
-            if ($token->getName() === $this->driver::UNKNOWN_TOKEN_NAME) {
-                $unknown[] = $token;
-                continue;
-            }
+            if (!$this->possibleTokensSearching) {
```

Comment:
Do I understand correctly that in this case this option is responsible for disabling exceptions if incorrect tokens are found?

If so, would it make sense to add the "throwOnError" flag? Or something like this?

Reply:
Yes, it is. For possible tokens it has to Parser work, so we can't stop on Lexer Exception.
Fix it by throwOnError flag adding.